site/docs/tutorials/snake/goal.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_label: Inspiration & Goal
+---
+
+# Inspiration and Goals
+
+Snake is a simple game. You can implement it in pretty any language or engine. It also allows for real-time user interaction and is a challenging game in itself.
+
+## Goals
+
+And the end of the tutorial, you will have a working version of snake.  
+You can expand on it and make your very own version along the way.
+
+During this tutorial you will learn how to ...
+
+- ... change the color palette
+- ... pick a color to draw
+- ... draw primitives (rectangles)
+- ... convert a sprite-image to source and draw it
+- ... handle user-input
+- ... publish your game
+
+## Preview
+
+At the end of this tutorial, your snake game will look like this:
+
+![Snake Preview](images/endresult.webp)
```

Comment:
I really like this page :smile: It's nice to have an overview of what will be built and take a peek at the final result.

Reply:
Yeah, I think it's important to know what you're going to create. And even more important: To know what you *won't* create.
People can get disappointed realy fast if they think they're building the next ultra-high FPS. ;-)